integration/docker/tag_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package docker
+
+import (
+	. "github.com/clearcontainers/tests"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("tag", func() {
+	var (
+		args     []string
+		id       string
+		exitCode int
+		tagName  string
+		command  *Command
+	)
+
+	runCommand := func(args []string, expectedExitCode int) {
```

Comment:
I noticed that you are using a similar pattern in other tests, would be nice to include it in a common file